From fa32003951787e5fb4e974d72700bfaf3ab1aa80 Mon Sep 17 00:00:00 2001
From: Ronny Chevalier <chevalier.ronny@gmail.com>
Date: Sun, 14 May 2017 13:19:11 +0200
Subject: [PATCH] test-conf-parser: add valid and invalid utf8 test for
 config_parse_path

(cherry picked from commit d5ade2d6027c67ef5a16d4a0e38c9fa38fd68212)
---
 src/test/test-conf-parser.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/test/test-conf-parser.c b/src/test/test-conf-parser.c
index 26ff27035b..77fcbc0dd3 100644
--- a/src/test/test-conf-parser.c
+++ b/src/test/test-conf-parser.c
@@ -109,8 +109,10 @@ static void test_config_parse_path(void) {
         test_config_parse_path_one("/path", "/path");
         test_config_parse_path_one("/path//////////", "/path");
         test_config_parse_path_one("///path/foo///bar////bar//", "/path/foo/bar/bar");
+        test_config_parse_path_one("/path/\xc3\x80", "/path/\xc3\x80");
 
         test_config_parse_path_one("not_absolute/path", NULL);
+        test_config_parse_path_one("/path/\xc3\x7f", NULL);
 }
 
 static void test_config_parse_log_level(void) {
